cmd/openshift-tests/upgrade.go

```diff
@@ -16,7 +16,7 @@ import (
 	"k8s.io/kubernetes/test/e2e/upgrades"
 )
 
-// upgradeSuites are all known upgrade test suites this binary should run
+// upgradeSuites are all known upgade test suites this binary should run
```

Comment:
mis-spelling

Reply:
Whoops, thanks!

Reply:
Oh I guess I fixed it in my commit, and reverting is removing it. :) This is probably ok, we'll revert the revert once I can get guidance from deads, and that will bring it back in.